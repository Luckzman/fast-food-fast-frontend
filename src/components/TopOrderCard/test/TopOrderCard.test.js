import React from 'react';
import { shallow } from 'enzyme';
import TopOrderCard from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<TopOrderCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
