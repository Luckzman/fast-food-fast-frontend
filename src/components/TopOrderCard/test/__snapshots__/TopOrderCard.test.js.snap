// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div
  className="order-card"
>
  <div
    className="order-img"
  >
    <img
      alt=""
    />
    <Link
      replace={false}
    >
      view details
    </Link>
  </div>
  <div
    className="card-content"
  >
    <h3 />
    <div
      className="star-rating"
    >
      <span
        className="fa fa-star checked"
      />
      <span
        className="fa fa-star checked"
      />
      <span
        className="fa fa-star checked"
      />
      <span
        className="fa fa-star checked"
      />
      <span
        className="fa fa-star checked"
      />
    </div>
    <p />
  </div>
</div>
`;
